src/plugins/image_display/ImageDisplay.cc

```diff
@@ -149,26 +149,74 @@ void ImageDisplay::LoadConfig(const tinyxml2::XMLElement *_pluginElem)
 void ImageDisplay::ProcessImage()
 {
   std::lock_guard<std::recursive_mutex> lock(this->dataPtr->imageMutex);
+
+  unsigned int height = this->dataPtr->imageMsg.height();
+  unsigned int width = this->dataPtr->imageMsg.width();
+  QImage::Format qFormat = QImage::Format_RGB888;
+
+  QImage image = QImage(width, height, qFormat);
+
+  common::Image::PixelFormatType pixelFormat =
```

Comment:
Can `pixelFormat` be declared below after the `switch` (line 192)?

Reply:
yep. 25eaf0c1d6233024dee32883fdb0447c7119de68